src/outputs/ascent.cpp

```diff
@@ -169,6 +170,11 @@ void AscentOutput::WriteOutputFile(Mesh *pm, ParameterInput *pin, SimTime *tm,
     auto &mbd = pmb->meshblock_data.Get();
 
     for (const auto &var : mbd->GetCellVariableVector()) {
+      // ensure that only cell vars are added (for now) as the topology above is only
+      // valid for cell centered vars
+      if (!var->IsSet(Metadata::Cell)) {
+        continue;
+      }
```

Comment:
This is the right fix even for the updates I am making to include face variables, etc. `GetCellVariableVector` is now just `GetVariableVector`.